doc/api/errors.md

```diff
@@ -500,7 +500,8 @@ of a string, it is the same as `error.code`.
 
 * {string}
 
-The `error.syscall` property is a string describing the [syscall][] that failed.
+The `error.syscall` property is a string describing the [`syscall(2)`][] that
+failed.
```

Comment:
This should link to http://man7.org/linux/man-pages/man2/syscalls.2.html rather than http://man7.org/linux/man-pages/man2/syscall.2.html – `syscall` itself is a specific function that is not really relevant to the concept of syscalls in general :)

Reply:
(i.e. we shouldn’t change the link text here either – we’re not talking about a code thing/identifier `syscall`)

Reply:
Amended. Please, check if it is correct now.